src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,34 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+
+  @media (min-width : 1024px) {
+    padding: 50px 40px;
+    width: calculate-width();
+    margin: 0 auto;
```

Comment:
there is no need to duplicate unchanging properties

Reply:
ok